docs/cli.md

```diff
@@ -49,6 +49,7 @@ Coder — A tool for provisioning self-hosted development environments with Terr
 | [<code>speedtest</code>](./cli/speedtest.md)           | Run upload and download tests from your machine to a workspace         |
 | [<code>ssh</code>](./cli/ssh.md)                       | Start a shell into a workspace                                         |
 | [<code>start</code>](./cli/start.md)                   | Start a workspace                                                      |
+| [<code>stat</code>](./cli/stat.md)                     | Show workspace resource usage.                                         |
```

Comment:
nit: Oh, we may want to add it also the site admin docs

Reply:
Probably the best place for this is in the agent-metadata docs.